packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/App.xaml

```diff
@@ -1,10 +1,10 @@
 <!-- Copyright (c) Microsoft Corporation. All rights reserved.
      Licensed under the MIT License.  -->
 <react:ReactApplication
-    x:Class="SampleApp.App"
+    x:Class="SampleAppCpp.App"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:SampleApp"
+    xmlns:local="using:SampleAppCpp"
```

Comment:
>SampleAppCpp [](start = 23, length = 12)

(nit) shouldn't this be called CppWinRt or something like that? It requires WinRt stack, right?

Reply:
Yes, we need WinRT stack, but the CppWinRt is quite different and we just want to say that this is a C++ code vs C# code.

---
In reply to: [359625669](https://github.com/microsoft/react-native-windows/pull/3802#discussion_r359625669) [](ancestors = 359625669)